code/modules/power/cell.dm

```diff
@@ -111,7 +111,7 @@
 	var/turf/T = get_turf(src.loc)
 	if (charge==0)
 		return
-	var/devastation_range = -1 //round(charge/11000)
+	var/devastation_range = CLAMP(round(sqrt(charge)/120),0,2)

```

Comment:
Just to be clear, `round` without a second arg is not normal rounding, it's flooring to the integer.